blog/_posts/2017-09-04-clojurians-update.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Clojurian's Update - Working on re-frame's Tracing Tool
```

Comment:
Hi, thanks for sending a PR for your 2nd blog post. Check out the comments below. :)

Reply:
thanks for reviewing, I'll make the adjustments in a bit :)